packages/rum-core/src/domain/tracing/tracer.spec.ts

```diff
@@ -75,8 +77,7 @@ describe('tracer', () => {
     })
 
     it("should trace request with priority '1' when sampled", () => {
-      spyOn(Math, 'random').and.callFake(() => 0)
-      const tracer = startTracer({ ...configuration, traceSampleRate: 50 }, sessionManager)
+      const tracer = startTracer({ ...configuration, traceSampleRate: 100 }, sessionManager)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏ Nice! No need to mock RND!